.circleci/config.yml

```diff
@@ -278,7 +278,7 @@ jobs:
         - run:
             name: Checkout the Content Repo
             command: |
-              git clone https://github.com/demisto/content.git
+              git clone https://github.com/demisto/content.git --branch server-items-list
```

Comment:
added for testing, will remove before merging

Reply:
remove before merging